mlir-clang/README.md

```diff
@@ -1,36 +1,42 @@
-##Build instructions
+## Build instructions
 
-        ## #Requirements -
-        working C and C++ toolchains(compiler, linker) - cmake - make
-    or ninja
+### Requirements 
+- Working C and C++ toolchains(compiler, linker)
+- cmake
+- make or ninja
 
-           ## #Process
-
-```
-#checkout the repository, 'cd' to it, then
-           mkdir
-           - p build cd build cmake../ llvm
-           - DLLVM_TARGETS_TO_BUILD =
-    "host;NVPTX" - DLLVM_ENABLE_PROJECTS =
-        "clang;mlir" - DLLVM_ENABLE_ASSERTIONS =
-            On - DCMAKE_BUILD_TYPE = Release - G Ninja #optional,
-             to use ninja instead of make make # or ninja; preferably with -j <num-cores>
+### 0. Clone Polygeist
+```sh
+git clone git@github.com:wsmoses/Polygeist.git $SOURCE_DIR
```

Comment:
I'd change this to be a clone via https. Also you can remote the submodule init/update if you make the clone recursive via `--recursive`.